testui/support/testui_images.py

```diff
@@ -99,20 +121,30 @@ def compare_images(original: str, comparison: str, threshold=0.9, image_match=''
     template = cv2.imread(comparison)
     image = cv2.imread(original)
     # loop over the scales of the image
-    threads = list()
+    threads = []
     parts = max_scale / 5.0
-    if min_scale > parts:
-        min_scale = parts
-    threads.append(threading.Thread(target=__compare, args=(
-        image, template, threshold, image_match, root_dir, parts, min_scale)))
-    threads.append(threading.Thread(target=__compare, args=(
-        image, template, threshold, image_match, root_dir, parts * 2.0, parts)))
-    threads.append(threading.Thread(target=__compare, args=(
-        image, template, threshold, image_match, root_dir, parts * 3.0, parts * 2.0)))
-    threads.append(threading.Thread(target=__compare, args=(
-        image, template, threshold, image_match, root_dir, parts * 4.0, parts * 3.0)))
-    threads.append(threading.Thread(target=__compare, args=(
-        image, template, threshold, image_match, root_dir, parts * 5.0, parts * 4.0)))
+    min_scale = min(min_scale, parts)
+
+    threads.append(
+        threading.Thread(target=__compare,
+                         args=(image, template, threshold, image_match,
+                               root_dir, parts, min_scale)))
+    threads.append(
+        threading.Thread(target=__compare,
+                         args=(image, template, threshold, image_match,
+                               root_dir, parts * 2.0, parts)))
+    threads.append(
+        threading.Thread(target=__compare,
+                         args=(image, template, threshold, image_match,
+                               root_dir, parts * 3.0, parts * 2.0)))
+    threads.append(
+        threading.Thread(target=__compare,
+                         args=(image, template, threshold, image_match,
+                               root_dir, parts * 4.0, parts * 3.0)))
+    threads.append(
```

Comment:
my preferred code format is one enforced by `black` formater.
```python
threads.append(
    threading.Thread(
        target=__compare,
        args=(
            image,
            template,
            threshold,
            image_match,
            root_dir,
            parts * 5.0,
            parts * 4.0,
        ),
    )
)
```
this code block, formated with `black` in my opinion becomes easier to read at a glance.

this is just a single example, the same preference applies to the whole py-test-ui repo. i think it would be valuable to discuss/vote on a default formatter and make it a requirement.